src/routes/(dashboard)/EditorDashboard.svelte

```diff
@@ -77,14 +82,23 @@
     // eslint-disable-next-line
     let table: Table<any> | null;
     $: $searchParams.sort && table?.resetScroll();
-    $: setTabContents($searchParams.tab, search);
+    $: setTabContents($searchParams.tab, search, $searchParams.project);
 
     const loadContents = async () => {
         [myWorkContents, myHistoryContents] = await Promise.all([
             data.editorDashboard!.assignedResourceContent.promise,
             data.editorDashboard!.assignedResourceHistoryContent.promise,
         ]);
     };
+
+    function projectNamesForContents(contents: ResourceAssignedToSelf[]) {
+        return Array.from(new Set(filterBoolean(contents.map((c) => c.projectName)))).sort();
+    }
+
+    function resetSelection() {
+        visibleMyWorkContents = [];
+        visibleMyHistoryContents = [];
+    }
```

Comment:
Does this need to be here? I thought this was only on the other tabs because the user could select items in the table.